.github/dependency-review-config.yml

```diff
@@ -0,0 +1,12 @@
+vulnerability_check: false
+allow_licenses:
```

Comment:
Can we preserve the commenting that we have in place previously? I think its helpful to see the categorization of the licenses